drivers/lora/sx1276.c

```diff
@@ -160,7 +174,25 @@ void SX1276SetAntSwLowPower(bool status)
 
 void SX1276SetBoardTcxo(u8_t state)
 {
-	/* TODO */
+#if DT_INST_NODE_HAS_PROP(0, tcxo_power_gpios)
+	bool enable = state;
+
+	if (enable == dev_data.tcxo_power_enabled) {
+		return;
+	}
+
+	dev_data.tcxo_power_enabled = enable;
```

Comment:
Move this statement below if/else condition

Reply:
Ok